MBBSEmu/HostProcess/ExportedModules/Majorbbs.cs

```diff
@@ -7532,5 +7542,43 @@ private void dcdate()
 
             Registers.AX = (ushort)((inputDateTime.Month << 5) + inputDateTime.Day + ((inputDateTime.Year - 1980) << 9));
         }
+
+        /// <summary>
+        ///     Determines if a Floating Point Co-Processor is present to handle x87 Instructions
+        ///
+        ///     Because the CpuCore in MBBSEmu supports x87, we'll return 3 which denotes its presence.
+        ///
+        ///     Signature: int _RTLENTRY _EXPDATA  _8087 = 3;
```

Comment:
might be nice to include the possible values here for posterity:

     0 - nocoprocessor, or ignored because of SET 87=N.
     1 - 8087 or 80187, or using coprocessor because of SET 87=Y.
     2 - 80287
     3 - 80387